installed-versions.json

```diff
@@ -97,8 +97,8 @@
       "overridden": false
     },
     "unplugin-vue-components": {
-      "version": "0.27.5",
-      "resolved": "https://registry.npmjs.org/unplugin-vue-components/-/unplugin-vue-components-0.27.5.tgz",
+      "version": "28.4.1",
```

Comment:
changed to https://antfu.me/posts/epoch-semver https://github.com/unplugin/unplugin-vue-components/releases

Reply:
Are there any tests for this?

Reply:
It ran on my machine